content/tos/index.md

```diff
@@ -0,0 +1,113 @@
+
++++
+title = ""
+template = "page.html"
++++
+
+<h1>Tlon Corporation<br />Urbit Hosting Services</h1>
+<h2>Terms of Service</h2>
+
+*Last Updated: September 8, 2020*
```

Comment:
Should this be updated to match merge date?

Reply:
It should be updated to reflect these changes, yes.